platform/linux-generic/include/odp_schedule_if.h

```diff
@@ -44,7 +53,7 @@ typedef void (*schedule_order_unlock_lock_fn_t)(void);
 typedef void (*schedule_order_lock_start_fn_t)(void);
 typedef void (*schedule_order_lock_wait_fn_t)(void);
 typedef uint32_t (*schedule_max_ordered_locks_fn_t)(void);
-typedef void (*schedule_save_context_fn_t)(uint32_t queue_index);
+typedef void (*schedule_config_fn_t)(schedule_config_t *config);
```

Comment:
Looks like `schedule_save_context_fn_t` is unused, so this deletion of a stale symbol should technically be a separate patch. This patch should just be adding `schedule_config_fn_t`. Minor point and probably not worrying about here.

Reply:
I think it would be wasteful (for review and git history) to create an extra patch for this removal of dead code while adding a new line next to it.